### ISSUE TITLE
Exercise 4 (propTypes) passes trivially

### ISSUE THREAD

**Issue by lastorset** [workshopper/learnyoureact#81]
Exercise 4 says to expect a `Warning` in the console, but no warning shows by default. This clause is added:

``` jsx
Todo.propTypes = {
        title: React.PropTypes.string.isRequired
    };
```

but both the `Todo`s already have titles.

One solution would be to add an invalid item in `TodoBox`

``` html
<Todo>5 km</Todo>
```

and add expected output, including a line like

``` html
<td style="border:1px solid black;">Run</td><td style="border:1px solid black;">5 km</td>
```

**Comment by kohei-takata** [comment]
I've wrongly changed `title: React.PropTypes.number.isRequired` to `title: React.PropTypes.string.isRequired`.
I merged #90 and return `string` to `number`, so that Exercise 4 seems to became more difficult :smile:

**Comment by mattez** [comment]
@kohei-takata where you changed `string` to `number`? There is still `string`.
